docs/application/dotnet/guides/nui/xaml/getting-started-with-nui-xaml.md

```diff
@@ -0,0 +1,114 @@
+# XAML Support for Tizen.NUI
+eXtensible Application Markup Language (XAML) allows you to define user interfaces in the Tizen.NUI applications using markup language rather than code. XAML is never required in a Tizen.NUI program, but it is often more succinct and more visually coherent than equivalent code, and potentially toolable. XAML is particularly well suited for using with the popular Model-View-ViewModel (MVVM) application architecture. XAML defines the View that is linked to the ViewModel code through the XAML-based data bindings.
+
+For more information, see https://docs.microsoft.com/en-us/xamarin/xamarin-forms/xaml/xaml-basics/
+
+XAML is mostly used to define the visual contents of a page and works together with a C# code-behind file. The code-behind file provides adding code logics for the markup.
+
+For more information, see https://docs.microsoft.com/en-us/xamarin/xamarin-forms/xaml/xaml-basics/get-started-with-xaml?tabs=windows
+
+## Create Solution
+To begin editing your first XAML file, use Visual Studio to create a new Tizen.NUI solution.
+
+For more information, see https://docs.microsoft.com/en-us/xamarin/xamarin-forms/xaml/xaml-basics/get-started-with-xaml?tabs=windows
```

Comment:
It would be better to use the page title than the URL.
I suggested to use as below:
```
For more information, see [Part 1. Getting Started with XAML](https://docs.microsoft.com/en-us/xamarin/xamarin-forms/xaml/xaml-basics/get-started-with-xaml?tabs=windows) in the XAML guide.
```

Reply:
I changed like this =>

For more information, see Part 1. Getting Started with XAML in the XAML guide.